### PR TITLE
Instagram Liker Bot

### PR DESCRIPTION
**Pull request by sayantani11**
I was assigned to create an Instagram liker bot i.e.  issue 
Fixes: #544 
Thank you for letting me work on it.
Hoping for a feedback soon.

### CODE REVIEW

File: Instagram Liker Bot/Instagram_Liker_Bot.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from selenium.webdriver.common.keys import Keys
+from getpass import getpass
+import time
+
+chrome=webdriver.Chrome(ChromeDriverManager().install())
+
+chrome.get("https://www.instagram.com")
+
+username=chrome.find_element_by_xpath('//*[@id="loginForm"]/div/div[1]/div/label/input')
+username.send_keys("") #enter username
```
Comment (kaustubhgupta) [inline Instagram Liker Bot/Instagram_Liker_Bot.py:12]: Take this as input before running the selenium part of the script

File: Instagram Liker Bot/Instagram_Liker_Bot.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.keys import Keys
+from getpass import getpass
```
Comment (kaustubhgupta) [inline Instagram Liker Bot/Instagram_Liker_Bot.py:4]: What is this getpass?

File: Instagram Liker Bot/Instagram_Liker_Bot.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+username.send_keys("") #enter username
+password= chrome.find_element_by_xpath('//*[@id="loginForm"]/div/div[2]/div/label/input')
+pswd=getpass()
+password.send_keys(pswd)
+login_button=chrome.find_element_by_xpath('//*[@id="loginForm"]/div/div[3]')
+login_button.click()
+time.sleep(10)
+
+search_bar= chrome.find_element_by_xpath('//*[@id="react-root"]/section/nav/div[2]/div/div/div[2]/input')
+search_bar.send_keys(" ") #enetr the username to be searched
```
Comment (kaustubhgupta) [inline Instagram Liker Bot/Instagram_Liker_Bot.py:21]: Take this as input before

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Add the readme file for this script

File: Instagram Liker Bot/Instagram_Liker_Bot.py (hunk 2 above)
Comment (sayantani11) [reply Instagram Liker Bot/Instagram_Liker_Bot.py:4]: It's for keeping the password encrypted

Comment (sayantani11) [reply Instagram Liker Bot/Instagram_Liker_Bot.py:4]: So that if the password is entered publicly it stays encrypted

**Comment by sayantani11** [comment]
Hi mentors 
Made all the changes that were required for #544 
Requesting for a feedback if any more change is required

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- How will I know that I need to enter my username or another person's useername first? You need to use some display statement for the inputs: 
```python
username.send_keys(input("Enter your username"))
```
- pswd=getpass() configure this also to display "Enter your password"

**Comment by sayantani11** [comment]
I made the required changes....Guess it should be fine now

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**